### PR TITLE
New: Add backup size information

### PR DESCRIPTION
**Pull request by ZackaryH8**
#### Database Migration
NO

#### Description
Added backup size information the the frontend
image

#### Todos
- [x] Tests
- [x] Wiki Updates

#### Issues Fixed or Closed by this PR

* #4830

### CODE REVIEW

File: frontend/src/System/Backup/BackupRow.js [hunk 1]
```diff
@@ -10,9 +10,9 @@ import TableRow from 'Components/Table/TableRow';
 import TableRowCell from 'Components/Table/Cells/TableRowCell';
 import RestoreBackupModalConnector from './RestoreBackupModalConnector';
 import styles from './BackupRow.css';
+import formatBytes from 'Utilities/Number/formatBytes';
```
Comment (markus101) [inline frontend/src/System/Backup/BackupRow.js:13]: This import should be directly under the React import.

File: frontend/src/System/Backup/BackupRow.js [hunk 2]
```diff
@@ -30,47 +30,35 @@ class BackupRow extends Component {
 
   onRestorePress = () => {
     this.setState({ isRestoreModalOpen: true });
-  }
+  };
```
Comment (markus101) [inline frontend/src/System/Backup/BackupRow.js:33]: A lot of unnecessary formatting here, please maintain the existing formatting.

File: frontend/src/System/Backup/Backups.js [hunk 3]
```diff
@@ -51,11 +55,11 @@ class Backups extends Component {
 
   onRestorePress = () => {
     this.setState({ isRestoreModalOpen: true });
-  }
+  };
```
Comment (markus101) [inline frontend/src/System/Backup/Backups.js:58]: Again more unnecessary formatting.

**Review by markus101 — CHANGES_REQUESTED** [review]
Mostly formatting changes that aren't needed and the one import needs to be moved.

File: frontend/src/System/Backup/BackupRow.js (hunk 2 above)
Comment (ZackaryH8) [reply frontend/src/System/Backup/BackupRow.js:33]: This was linted using the script in the package.json

Comment (markus101) [reply frontend/src/System/Backup/BackupRow.js:33]: It passes linting, but in this case it's about keeping the consistency with rest of the repo. It looks like prettier (or similar) reformatted these lines.

**Review by markus101 — CHANGES_REQUESTED** [review]
Still a bunch of formatting changes, please revert.
